### DISCUSSION TITLE
Change font-size in parts of the number individually

### DISCUSSION THREAD

**Discussion by ismamz** [barvian/number-flow#21] [Example request]
How do I change the `font-size` for the `$` symbol and the decimal part of the number? (without using framer-motion).

For example: `$20.25`

```tsx
<NumberFlow
  value={20.25}
  format={{
    style: 'currency',
    currency: 'USD',
    minimumFractionDigits: 2,
    maximumFractionDigits: 2,
  }}
  locales="en-US"
/>
```

**Comment by barvian** [comment]
Hey, I'd definitely like to support this at some point but it introduces some complexity with text selection and the mask-image that I haven't figured out great solutions to yet 🤔
